lib/season_stats.rb

```diff
@@ -4,8 +4,7 @@ def winningest_coach(season_id)
         get_all_games(season_id)
         load_coach_array
         load_coach_hash
-        coach_percentage_hash
-        @coach_percentages.sort_by {|k, v| -v}.first[0]
+        coach_percentage_hash.sort_by {|k, v| -v}.first[0]
```

Comment:
Looks great James. I see what Chee is saying but you argument about not messing with any class files outside of the module makes sense as well.